src/Run/SettingValueProvider.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Microsoft.DotNet.Execute
+{
+    internal static class SettingValueProvider
+    {
+        private static string s_cpuArch = null;
+        private static string s_osName = null;
+        private static string s_osRid = null;
+        private static Dictionary<string, string> s_osToDefaultVersionMap = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
+        {
+            { OS.OSX, "osx.10.10" },
+            { OS.FreeBSD, "osx.10.10" },
```

Comment:
In corefx, build.sh is assinging osx.10.10 as the default values for testnugetruntimeid..